Replicator/Puller.cc

```diff
@@ -356,21 +356,59 @@ namespace litecore::repl {
         }
     }
 
-    Worker::ActivityLevel Puller::computeActivityLevel() const {
-        ActivityLevel level;
+    Worker::ActivityLevel Puller::computeActivityLevel(std::string* reason) const {
+        ActivityLevel         level;
+        int                   k = -1;
```

Comment:
....a variable named `k` ?  Name it something useful please....

Reply:
I could not think a meaning name. Perhaps name it, decisionPlace, or decisionPoint?

Reply:
Probably because it is meaningless, as I pointed out in another comment about not separating this into two steps.